priv/repo/migrations/20220331123334_add_json_to_saas_table.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Glific.Repo.Migrations.AddJsonToSaasTable do
```

Comment:
@AkhileshNegi let's change this file and merge this.

Reply:
made requested changes